frontend-react/src/content/product/index.tsx

```diff
@@ -0,0 +1,48 @@
+/* Data that drives breadcrumb creation and slug appending */
```

Comment:
[nit] if the idea with these index files is that they are creating the scaffolding that the .tsx files in the /pages directory will use to render the pages, but not actually containing any render logic themselves, then for the sake of clarity (and nothing else because functionally who cares), it seems like these should be .ts files

Reply:
This one could probably by a ts file for now. Take a look in the one for Resources, though, where it uses a wrapper function that takes a component in JSX form. 🤔 I'll see if I can remove the need for that function there and maybe use it somewhere in the template instead but that was the original need for the tsx extension. 

If I can't, I'll have to leave it as is but that does kinda blend into the layout logic imo

Reply:
I ended up altering the function taking JSX to take either `() => JSX.Element` OR `JSX.Element` so either can be passed in and rendered with crumbs. That removed the need for any JSX syntax in these files. I couldn't, however, make the wrapper a part of the rendering process instead of this instantiation process.